xml/ftp_banners.xml

```diff
@@ -808,4 +808,17 @@ against these patterns to fingerprint FTP servers.
     <param pos="1" name="host.name"/>
     <param pos="2" name="service.version"/>
   </fingerprint>
+  <fingerprint pattern="^Welcome to the (?:Cisco )?(?:Tele[Pp]resence) ([a-zA-Z\s]*?) ((?:MSE )?\d+), version (\d+.\d+\(\d+.\d+\)).*?">
```

Comment:
Rather than [Pp] and [a-zA-Z], you can either make the entire fingerprint case insensitive (with a ```flags``` attribute). Alternatively, you can make just some parts of it case insensitive, like:

```
(?i:Telepresence)
```